content/docs/contributing/style_guide.md

```diff
@@ -32,23 +32,53 @@ When it is advantageous, data should be [stored in data files and processed usin
 
 ## Style Enforcement
 
-Markdown styling is enforced by [markdownlint](https://github.com/markdownlint/markdownlint) using the configuration {{% repo_link path=".mdl_style.rb" text=".mdl_style.rb" %}}.
-An explanation of the rules can be found [here](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md).
+Markdown styling is enforced by [`markdownlint`](https://github.com/markdownlint/markdownlint) using the configuration {{% repo_link path=".mdl_style.rb" text=".mdl_style.rb" %}}.
+An explanation of the style rules can be found [here](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md).
 
-You can optionally install markdownlint locally to check your changes before submitting a pull request.
+You can optionally install `markdownlint` locally to check your changes before submitting a pull request.
+
+### Using `pre-commit`
+
+The easiest way to do this is to use the [pre-commit](https://pre-commit.com/) tool, which has already been set up in the handbook repository.
+
+```bash
+brew install pre-commit   # if not already installed
+pre-commit run markdownlint --all-files
+```
+
+`pre-commit` is intended to be run before all commits (to make sure that you don't unintentionally commit something unwanted).
+To see how to set this up, see the [Contributing Changes]({{< ref "/content/docs/contributing/contributing_changes.md#making-changes" >}}) page.
+
+### Manual Installation
 
 {{% hint info %}}
-Make sure you install the Ruby version (`mdl`) and not the Node.js version (`markdownlint-cli`) contained in the homebrew repositories,
+Make sure you install the Ruby version (`mdl`) and not the Node.js version (`markdownlint-cli`, which the Homebrew formula provides),
 since the configuration is only good for the former.
 {{% /hint %}}
 
-Once installed and assuming you're in the repository root, you can lint a specific file you've made changes to like this:
+First, install Ruby with Homebrew, and then use that to install `mdl`.
+macOS actually provides a system installation of Ruby, but using that isn't advisable as you need to install gems as the root user.
```

Comment:
I'm not sure if this is true. Using the system `gem` packages I install as a normal user go to `$GEM_HOME`.

Reply:
As above I think we might just delete this block, so the rest is a bit of a digression (feel free to ignore). I've almost no experience with ruby so I might be wrong, but this was my experience with system `ruby` and `gem`.

@llewelld Sorry, I can't remember — did you mention you managed to install `mdl` with system `gem`?

```bash
$ which -a gem
/opt/homebrew/opt/ruby/bin/gem
/usr/bin/gem

$ /usr/bin/gem install mdl
Fetching chef-utils-18.1.29.gem
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.6.0 directory.

$ echo $GEM_HOME


$ sudo /usr/bin/gem install mdl
Password:
Fetching concurrent-ruby-1.2.2.gem
Fetching mixlib-cli-2.1.8.gem
Fetching tomlrb-2.0.3.gem
Fetching mixlib-config-3.0.27.gem
Fetching chef-utils-18.1.29.gem
Fetching kramdown-2.4.0.gem
Fetching mixlib-shellout-3.2.7.gem
Fetching kramdown-parser-gfm-1.1.0.gem
Fetching mdl-0.12.0.gem
Successfully installed kramdown-2.4.0
Successfully installed kramdown-parser-gfm-1.1.0
Successfully installed mixlib-cli-2.1.8
Successfully installed tomlrb-2.0.3
Successfully installed mixlib-config-3.0.27
Successfully installed concurrent-ruby-1.2.2
Successfully installed chef-utils-18.1.29
Successfully installed mixlib-shellout-3.2.7
ERROR:  Error installing mdl:
	The last version of mdl (>= 0) to support your Ruby & RubyGems was 0.11.0. Try installing it with `gem install mdl -v 0.11.0`
	mdl requires Ruby version >= 2.7. The current ruby version is 2.6.10.210.
```